lib/Installation/ModuleRegistration/ModuleRegCodeController.pm

```diff
@@ -0,0 +1,60 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class introduces business actions for Module Registration Code dialog.
+#
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::ModuleRegistration::ModuleRegCodeController;
+use strict;
+use warnings;
+use Installation::ModuleRegistration::ModuleRegCodePage;
+use Installation::Popups::UntrustedGPGPopup;
+use YuiRestClient;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {}, $class;
+    return $self->init($args);
+}
+
+sub init {
+    my ($self, $args) = @_;
+    $self->{ModuleRegCodePage} = Installation::ModuleRegistration::ModuleRegCodePage->new({app => YuiRestClient::get_app()});
+    $self->{UntrustedGPGKeyPopup} = Installation::Popups::UntrustedGPGPopup->new({app => YuiRestClient::get_app()});
+    return $self;
+}
+
+sub get_module_regcode_page {
+    my ($self) = @_;
+    die "Module Registration Code page is not displayed" unless $self->{ModuleRegCodePage}->is_shown();
```

Comment:
please use the full description on the page: "Extension and Module Registration Codes page"